sentinel-ai-session/src/main/java/com/phonepe/sentinel/session/history/HistoryStore.java

```diff
@@ -0,0 +1,10 @@
+package com.phonepe.sentinel.session.history;
+
+import java.util.Optional;
+
+public interface HistoryStore {
```

Comment:
JavaDoc on everything

Reply:
Aslo can this not be merged into SessionStore itself? Seems like related functionality...?